R/data.R

```diff
@@ -0,0 +1,41 @@
+#' Dataset correspondence table between country/study site codes, country/study site names
+#'
+#' Dataframe providing the correspondence table of the id code \code{ID}
+#' and the geographical label \code{Label} to use
+#' throughout reports. Additional information is available in columns 
+#' \code{LocationName},
+#' \code{LocationType},
+#' and \code{CountryISO2Code}
+#' Original Excel spreadsheet \code{suggestion1_RefLocation.xlsx} available on GoogleDrive.
+#'
+#' @format A data frame with 40 rows and 5 variables:
+#' \describe{
+#'   \item{ID}{Code associated to a location (country or study site), e.g. AT, BE, BG, etc.}
+#'   \item{Label}{Full name of the country or study site including 
+#'   'the' article for NL and UK  e.g. Austria, Belgium, the Netherlands, the United Kingdom etc.}
+#'   \item{LocationName}{Shorter name of the country or study site}
+#'   \item{LocationType}{For location, variable specifying the goegraphical level, i.e., country or Nuts3 location}
+#'   \item{CountryISO2Code}{ISO2 code of the corresponding country, e.g. ES for Navarra}
+#' }
+#' 
+"RefSite"
+
+#' Dummy dataset
+#'
+#' Dataframe with commun epidemiological variables to use for tests
+#'
+#' @format A data frame with 100 observations on 8 variables.
+#' \describe{
+#'   \item{id}{Identifier}
```

Comment:
Sorry Cristina I missed this! 
If we move to CamelCase, should we put ID, Country, Age and Case?
Let me know your thoughts! :)

Reply:
i have not idea about this... maybe @epi-gde can confirm? But sure I can change it!

Reply:
Hi Cristina! :)
I have upgraded the relase number to v0.5.1.0, as agreed with Gilles :)
Let me know what you decide for the column label! In any case, I would be in favor of using the same as with RefStie. But I am happy to change if you prefer the other spelling!